packages/web-components/src/wallet-connection/watchWallet.ts

```diff
@@ -260,8 +243,10 @@ export const watchWallet = (
 
       for await (const update of iterateEach(follower)) {
         console.debug('wallet update', update);
+        // @ts-expect-error xxx
```

Comment:
Just wondering, what was the error here? Maybe we can provide a type for the follower, or `iterateEach<>`? Or casting doesn't support it well enough..